docs/index.tpl.html

```diff
@@ -407,11 +406,22 @@ <h1>CSS Variables</h1>
 				<td><code>--mv-item-height</code></td>
 				<td><a href="/docs/collections/#collectiion-editing-controls">Item bar</a> of <a href="/docs/collections/#simple-collections">simple collections</a></td>
 				<td>Height of item in px, e.g. <code>100px</code>.</td>
-			</tr>
+			</tr> -->
 		</tbody>
 	</table>
+
+
 </section>
 
+<div hidden>
+	<select property="live" id="live">
+		<option value="✘">✘</option>
+		<option value="✓">✓</option>
+		<option value="Initially">Initially</option>
+		<option value="Expressions only">Expressions only</option>
```

Comment:
You don't need to set the value property when it's the same as the content of the `<option>`, and it's not [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) either.